<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QMainWindow" name="RegistrationWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrierung DiaThesis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="nutzertyp_page">
         <widget class="QGroupBox" name="nutzertyp_gb">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>60</y>
            <width>221</width>
            <height>141</height>
           </rect>
          </property>
          <property name="title">
           <string>Ich möchte die Applikation nutzen als:...</string>
          </property>
          <widget class="QRadioButton" name="Arzt_rb">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>82</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Arzt</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="Patient_rb">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>70</y>
             <width>82</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Patient</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="Angehoeriger_rb">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>110</y>
             <width>82</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Angehöriger</string>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="seiteZweiReg_btn">
          <property name="geometry">
           <rect>
            <x>754</x>
            <y>512</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>weiter</string>
          </property>
         </widget>
         <widget class="QPushButton" name="zurAnmeldung_btn">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>520</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>zurück zur Anmeldung</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="persoenlDaten_gb">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>220</y>
            <width>341</width>
            <height>221</height>
           </rect>
          </property>
          <property name="title">
           <string>Persönliche Daten: </string>
          </property>
          <widget class="QLabel" name="vorname_lb">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>Vorname: </string>
           </property>
          </widget>
          <widget class="QLabel" name="nachname_lb">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Nachname: </string>
           </property>
          </widget>
          <widget class="QLineEdit" name="vorname_le">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>40</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>Max</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="nachname_le">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>100</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>Mustermann</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="geburtstag_le">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>170</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>YYYY.MM.DD</string>
           </property>
          </widget>
          <widget class="QLabel" name="geburtstag_lb">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Geburtsdatum:</string>
           </property>
          </widget>
          <widget class="QLabel" name="geburtstagTip_lb">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>170</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>(YYYY.MM.DD)</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="MailPW_page">
         <widget class="QPushButton" name="accountErstellen_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>750</x>
            <y>510</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Account erstellen</string>
          </property>
         </widget>
         <widget class="QPushButton" name="zurSeite1Reg_btn">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>510</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>zurück</string>
          </property>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>100</y>
            <width>331</width>
            <height>211</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLineEdit" name="user_le">
             <property name="placeholderText">
              <string>example@host.com</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="password2_le">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>********</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="password_le">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>********</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="mail_lb">
             <property name="text">
              <string>Email: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="passwort_lb">
             <property name="text">
              <string>Passwort:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="passwort2_lb">
             <property name="text">
              <string>Passwort bestätigen:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>340</y>
            <width>331</width>
            <height>141</height>
           </rect>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>322</width>
             <height>141</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QCheckBox" name="agb_cb">
              <property name="text">
               <string>Ich habe die AGBs gelesen und akzeptiere sie</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="datenschutz_cb">
              <property name="text">
               <string>Ich habe die Datenschutzerklärung gelesen und akzeptiere sie</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QLabel" name="passwortSicherheit_lb">
          <property name="geometry">
           <rect>
            <x>690</x>
            <y>180</y>
            <width>221</width>
            <height>151</height>
           </rect>
          </property>
          <property name="text">
           <string>Passwortsicherheit:


Ein sicheres Passwort besteht aus: 

 mindestens 6 Zeichen
 mindestens einem Kleinbuchstaben
 mindestens einem Großbuchstaben
 mindestens einer Zahl</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
         <widget class="QLabel" name="PwHint_lb">
          <property name="geometry">
           <rect>
            <x>690</x>
            <y>200</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
